On Windows Illegal char :

**Describe the bug**
Can't create export file with colons on windows.

**To Reproduce**
Run app on windows OS.

**Expected behavior**
Export file should be created without errors. Probably should change colons to something else.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - SDK: jdk-17.0.2

**Additional context**
Caused by: java.nio.file.InvalidPathException: Illegal char <:> at index 76:

--- a/README.md
+++ b/README.md
@@ -26,6 +26,14 @@ Added integration with external services for word translation and definition loo
 
 ### Usage
 
+Program will look up every word in GT for translation, transcription and definitions (with default
+options), and dictionaryapi (if enabled) for definitions, synonyms, antonyms and examples and
+transcription. If word in dictionary api is found then only this data is used. If word in dictionary
+api is not found, then data from GT is used. If none found, then meanings are empty, because GT at
+least has a translation.    
+GT can translate from any language to any language, but dictionary api works only with english
+source language.
+
 #### Available options of jar startup
 
     * korvo-to-anki.api.dictionaryApiEnabled: true/false - turn on or off dictionaryapi service integration. Default - true
@@ -64,12 +72,14 @@ OPTIONS
 
 `ae --sourceLanguage ja --targetLanguage en --options t,at,md,rm`
 
-Options t,at,md,rm are the only options that currently works. These options are google translate options.
+Options t,at,md,rm are the only options that currently works. These options are google translate
+options.
 
 #### Run result
 
 The result of run is going to be stored in the same folder as jar with name -
-**korvo-to-anki-TIMESTAMP.txt**. Also creates **korvo-to-anki.state** file with the json of already exported
+**korvo-to-anki-TIMESTAMP.txt**. Also creates **korvo-to-anki.state** file with the json of already
+exported
 words. On new run will check for state file and filter already exported words from new run.
 
 #### Show definitions in cards

--- a/build.gradle
+++ b/build.gradle
@@ -5,7 +5,7 @@ plugins {
 }
 
 group = 'ru.dankoy'
-version = '0.1.3-SNAPSHOT'
+version = '0.1.4-SNAPSHOT'
 
 java {
     sourceCompatibility = '17'

--- a/src/main/java/ru/dankoy/korvotoanki/core/service/datetimeprovider/DateTimeProviderImpl.java
+++ b/src/main/java/ru/dankoy/korvotoanki/core/service/datetimeprovider/DateTimeProviderImpl.java
@@ -13,7 +13,7 @@ public class DateTimeProviderImpl implements DateTimeProvider {
   @Override
   public String now() {
 
-    var format = "yyyy-MM-dd_HH:mm:ss";
+    var format = "yyyy-MM-dd_HH-mm-ss";
 
     var localDateTime = LocalDateTime.now();
 

--- a/src/test/java/ru/dankoy/korvotoanki/core/service/datetimeprovider/DateTimeProviderImplTest.java
+++ b/src/test/java/ru/dankoy/korvotoanki/core/service/datetimeprovider/DateTimeProviderImplTest.java
@@ -24,7 +24,7 @@ class DateTimeProviderImplTest {
   @Test
   void now() {
 
-    var format = "yyyy-MM-dd_HH:mm:ss";
+    var format = "yyyy-MM-dd_HH-mm-ss";
     var ld = LocalDate.of(1989, 1, 13);
     var lt = LocalTime.of(10, 10, 10);
     var ldt = LocalDateTime.of(ld, lt);

